tests/parser/tests/test_runner.rs

```diff
@@ -15,8 +15,14 @@ use hash_utils_testing_macros::generate_tests;
 use lazy_static::lazy_static;
 use regex::Regex;
 
+extern crate static_assertions;
+
 /// Whether or not the UI tests should re-generate the output.
-const REGENERATE_OUTPUT: bool = true;
+const REGENERATE_OUTPUT: bool = false;
+
+// Assert that `REGENERATE_OUTPUT` isn't set to `true` in normal running
+// conditions
+static_assertions::const_assert!(!REGENERATE_OUTPUT);
```

Comment:
How is this not hit when one wants to regenerate output?

Reply:
This should hit it and alert that the tests are set to `pass` all the time, this essentially prevents it from being set to `true` when it really shouldn't

Reply:
But in order to regenerate output one has to comment out this line first?

Reply:
Ah yes of course, okay Ill put the assertion at the end of the tests